torch/_library/custom_ops.py

```diff
@@ -392,12 +392,12 @@ def fake_impl(*args, **kwargs):
         lib._register_fake(self._name, fake_impl)
 
         autograd_impl = _library.autograd.make_autograd_impl(self._opoverload, self)
-        lib.impl(self._name, autograd_impl, "Autograd")
+        lib.impl(self._name, autograd_impl, "Autograd", dispatcher_convention=True)
```

Comment:
Other naming proposals:
- with_keyset
- redispatch_convention
- pass_keyset

Also should we do what we do in c++ where we inspect the first argument name and if it is name keyset, we use this convention automatically? :p

Reply:
> Also should we do what we do in c++ where we inspect the first argument name and if it is name keyset, we use this convention automatically? :p

I'm happy to do this, was worried about if it was too BC-breaking.

Reply:
> Also should we do what we do in c++ where we inspect the first argument name and if it is name keyset, we use this convention automatically? :p

I changed my mind, this is risky because inspect.signature doesn't handle all cases so we'd likely break BC.